pkg/watchdog/init.sh

```diff
@@ -36,7 +43,7 @@ run_watchdog() {
       # Now lets see if we need to rebuild the configuration file
       (cat /etc/watchdog.conf.seed
        find /run/watchdog -type f | sed -e 's#/run/watchdog/pid#pidfile = /run#' \
-          -e '/\/run\/watchdog\/file/a change = 300'                             \
+          -e "/\/run\/watchdog\/file/a change = $WATCHDOG_CHANGE_TIME"                             \
```

Comment:
Changing this to more than 300 means we risk getting close to the default test time after an eveimage update (which is 600 seconds). If that is done we might commit to an EVE image which will not work.

Reply:
To add to @eriknordmark point -- lets remove the watchdog change from this PR altogether.

It really doesn't belong here (since this one is about cgroups)

Adarsh, once you do that -- I'll be ready to do a final review and merge this

Reply:
Removed.